go/vt/vtctl/vtctl.go

```diff
@@ -2311,17 +2317,25 @@ func commandGetSchema(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag
 		excludeTableArray = strings.Split(*excludeTables, ",")
 	}
 
-	sd, err := wr.GetSchema(ctx, tabletAlias, tableArray, excludeTableArray, *includeViews)
+	resp, err := wr.VtctldServer().GetSchema(ctx, &vtctldatapb.GetSchemaRequest{
```

Comment:
calling this out as a code change in the legacy server, which _is_ backwards compatible, but technically a change